src/actions/rotate/Rotate.ts

```diff
@@ -0,0 +1,42 @@
+import horizontalFlip from "./horizontalFlip";
+import verticalFlip from "./verticalFlip";
+import angle from "./angle";
+import Action from "../Action";
+import {IAction} from "../../interfaces/IAction";
+import Param from "../../parameters/Param";
+
+class Rotate extends Action implements IAction {
+  constructor(angle?: number) {
+    super();
+    this.addParam(new Param('a', angle));
+  }
+
+  /**
+   * @return {Rotate}
+   */
+  horizontalFlip(): Rotate {
+    this.params.get('a').addValue('hflip');
```

Comment:
This feels a bit strange,
For addParam, we have a proper method.
however to reach down into the params itself to get a key.

Can we add a function to getParamByKey? do you think it makes sense?

Reply:
done